deploy/handler/cluster_role.yaml

```diff
@@ -0,0 +1,19 @@
+{{if .ClusterReaderExists}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nmstate-cluster-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
+rules:
+- apiGroups:
+  - nmstate.io
+  resources:
+  - nodenetworkstates
+  - nodenetworkconfigurationpolicies
+  - nodenetworkconfigurationenactments
```

Comment:
Adding `nmstates` (`nmstates.nmstate.io`) to this role could make sense too so we have all the nmstate.io resources, or WDYT?

Reply:
It seems that we can already read it anyhow by the cluster-reader
so do we still want to add it ? (it would be duplicate, unless we remove it from somewhere else that allows it)

Reply:
hmm. ok. ack